leaves/burndown.go

```diff
@@ -714,7 +714,7 @@ func (analyser *BurndownAnalysis) MergeResults(
 // least of (sampling1, sampling2) and (granularity1, granularity2).
 func mergeMatrices(m1, m2 DenseHistory, granularity1, sampling1, granularity2, sampling2 int,
 	c1, c2 *core.CommonAnalysisResult) DenseHistory {
-	commonMerged := *c1
+	commonMerged := c1.Copy()
```

Comment:
Is c1 at some point written in parallel? Because if it is, you would still need to wrap the reading and writing in a mutex.

Reply:
There is a mutex at the higher level for the rest of the code. This piece expects an immutable object, and I forgot about the mutable maps.

So answering the question: no, it is not, and not intended to be possible.